packages/atomic/src/utils/replace-breakpoint.ts

```diff
@@ -33,7 +35,7 @@ function replateStyleContent(element: HTMLElement, breakpoint: string) {
 }
 
 export function updateBreakpoints(element: HTMLElement) {
-  const layout = element.closest('atomic-search-layout');
+  const layout = closest(element, 'atomic-search-layout');
```

Comment:
The other issue with `updateBreakpoints` was that, when looking for a parent `atomic-search-layout`, it didn't traverse the ShadowDOM. I introduced a util that can traverse the ShadowDOM.